src/new-client/src/services/status.ts

```diff
@@ -0,0 +1,38 @@
+import { watch$ } from "services/client"
+import { scan } from "rxjs/operators"
+
+export interface RawServiceStatus {
+  Type: string
+  Instance: string
+  TimeStamp: number
+  Load: number
+}
+
+export interface ServiceInstanceStatus {
+  serviceType: string
+  serviceId: string
+  timestamp: number
+  serviceLoad: number
+  isConnected: boolean
+}
+
+const convertFromRawMessage = (
+  serviceStatus: RawServiceStatus,
+): ServiceInstanceStatus => {
+  return {
+    serviceType: serviceStatus.Type,
+    serviceId: serviceStatus.Instance,
+    timestamp: serviceStatus.TimeStamp,
+    serviceLoad: serviceStatus.Load,
+    isConnected: true,
```

Comment:
I don't see the point of adding the `isConnected` property, given the fact that its value will always be `true` looks misleading...